assets/dev/js/editor/regions/responsive-bar/view.js

```diff
@@ -168,6 +171,22 @@ export default class View extends Marionette.ItemView {
 		this.ui.sizeInputHeight.val( Math.round( size.height ) );
 	}
 
+	onPreviewOpen() {
+		this.setWidthHeightInputsEditableState();
```

Comment:
Why does it happen on preview open? It should be done on every device mode change

Reply:
It needs to happen both on device-mode-change and also on preview-open.
Because when you click on the "responsive-mode" button from the panel, the event of "on device-mode-change" is not triggered.